### PR TITLE
fix: remove unnecessary fields and add populate the createdBy for compositions

### PR DESCRIPTION
**Pull request by JivusAyrus**
- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

**Review by StarpTech — APPROVED** [review]
LGTM

File: controlplane/src/bin/version-migration.ts
Comment (StarpTech) [inline controlplane/src/bin/version-migration.ts:1]: Don't forget to delete script in package.json

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: controlplane/src/bin/version-migration.ts
Comment (JivusAyrus) [reply controlplane/src/bin/version-migration.ts:1]: Done

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**